GitCommands/StringPool.cs

```diff
@@ -178,44 +178,46 @@ internal static unsafe bool EqualsAtIndex(string source, int index, string compa
             }
 
             fixed (char* pc = comparand)
-            fixed (char* ps = source)
```

Comment:
Is this just an indentation change? Perhaps revert, or add braces.

Reply:
Braces are there, I'm guessing you've missed them.
Same for the other one.